api/models/product.py

```diff
@@ -0,0 +1,27 @@
+from models.base import Base
+from typing import List
+from sqlalchemy import String, Numeric
+from sqlalchemy.orm import Mapped, mapped_column, relationship
+from decimal import Decimal
+
+class Product(Base):
+
+    __tablename__ = "products"
+
+    id: Mapped[int] = mapped_column(primary_key=True)
+    name: Mapped[str] = mapped_column(String(30), nullable=False)
+    description: Mapped[str] = mapped_column(String(50))
```

Comment:
I think `50` is such a small number for a product description, I guess maybe `255` would be fine.